README.md

```diff
@@ -1,6 +1,8 @@
+[![Build Status](https://travis-ci.com/elheck/Squiddrone.svg?branch=dev)](https://travis-ci.com/elheck/Squiddrone)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Could not find module `remark-lint-no-unneeded-full-reference-image`](https://app.codacy.com/manual/SquidSquad/Squiddrone/pullRequest?prid=4738136)

Reply:
Seems to be a bug of RemarkLint of Codacy. Will be ignored for now